tensorflow/go/saved_model.go

```diff
@@ -45,7 +45,7 @@ type SavedModel struct {
 // exported in other languages, such as using tf.saved_model.builder in Python.
 // See:
 // https://www.tensorflow.org/code/tensorflow/python/saved_model/
-func LoadSavedModel(exportDir string, tags []string, options *SessionOptions) (*SavedModel, error) {
+func LoadSavedModel(exportDir string, tags []string, options *SessionOptions, runOptions *[]byte, metaGraph *[]byte) (*SavedModel, error) {
```

Comment:
Two things here:

(1) It feels like it would be better for `metaGraph` to be a field of the `SavedModel` structure instead of being an "output parameter" to `LoadSavedModel`.

(2) The signature here is a bit awkward, I'm thinking of some alternatives. Though, before that, am curious about the use case you have for providing custom run options to `LoadSavedModel`. Mind elaborating on that? (Note that these options only apply to the underlying `Session.run` call to restore the model parameters from the checkpoint, am curious which options you found the need to set)

Reply:
@asimshankar  thank you very much for reviewing.
It seems that the signature was a little confusing when I used pointers on `runOptions` and `metaGraph`. For now it's just passed.
Actually I would like to have not `[]byte`, just some concrete types for this. As for example we have in the python version.